data/saas/saas_connector_registry.toml

```diff
@@ -16,6 +16,12 @@ dataset = "data/saas/dataset/datadog_dataset.yml"
 icon = "data/saas/icon/default.svg"
 human_readable = "Datadog"
 
+[firebase_auth]
+config = "data/saas/config/firebase_auth_config.yml"
+dataset = "data/saas/dataset/firebase_auth_dataset.yml"
+icon = "data/saas/icon/default.svg"
```

Comment:
not sure if we want to put an image in here or just use the default...

Reply:
don't think this is important atm